shell/utils/axios.js

```diff
@@ -1,4 +1,5 @@
-import Axios from 'axios';
+import Axios, { CancelToken, isCancel } from 'axios';
```

Comment:
Are these changes still needed now that axios is not being bumped in this PR?

Reply:
Not needed and removed. Also updated the PR description.